lib/task_5/task5.dart

```diff
@@ -0,0 +1,33 @@
+// Есть коллекция строк вида ‘one, two, three, cat, dog’ или любого другого.
+// Реализуйте метод, возвращающий цифры без повторений, которые встречаются в
+// данной строке. Однако цифры встречаются в виде английских слов от zero до nine.
+// Например, в результате строки ‘one, two, zero, zero’ мы получим следующий
+// результат: [1, 2, 0]. Если в строке есть слова, не являющиеся цифрами
+// от 0 до 9, пропускайте их.
+
+Set<int?> numbersWithoutRepetitions(List<String> wordsCollection) {
```

Comment:
```suggestion
Set<int> numbersWithoutRepetitions(List<String> wordsCollection) {
```

У нас не предполагается, что цифра может быть null.

Reply:
Android Studio против этого! И я на ее стороне. Может я что то не понял?

Reply:
У нас по прежнему не предполагается, что цифра может быть null.
Если цифры нет, то просто не вносим его в Set.

Убери знак вопроса и в цикле сделай проверку на то mapNumber[word] не null.